docs/development-update-22.44.md

```diff
@@ -0,0 +1,120 @@
+# Development Update 22.44
+
+## Overview
+
+This is an interim update ("code drop") to give the validation team
+an opportunity to take the DPDK code out for a test drive.
+
+## Project Name
+
+Note that the P4-OVS Split Architecture has a new name: **P4 Control Plane**.
+
+## What's Changed
+
+### Highlights
+
+- Restructured Stratum to support TDI variants
+- Integrated DPDK support from P4-OVS
+- Disabled unsupported port attributes in DPDK
+- Updated OVS to newer release
+- Replace `gnmi_cli` with `gnmi-ctl` in DPDK build
```

Comment:
Replace ovs-p4ctl with p4rt-ctl?

Reply:
We did this in September, as part of the Tofino release. It's unchanged in this drop.